tests/clear_backends_test.py

```diff
@@ -22,6 +24,10 @@
 
 config.parse_flags_with_absl()
 
+with contextlib.suppress(ImportError):
+  import pytest
+  pytestmark = pytest.mark.pjrt_c_api_unimplemented  # crashes runtime
```

Comment:
What if you make the new `pytest_mark_if_available` function into a class with a `__call__` and `__getattr__` defined, so that here and elsewhere you could just write
```python
pytestmark = jtu.pytest_mark_if_available.pjrt_c_api_unimplemented  # crashes runtime
```

Reply:
Ah I see the global variable works: https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules

This kind of magic is one of the downsides of pytest IMO... how would you feel about using it as a class decorator here instead?

Reply:
Switched to using class decorator, see other comment.